test/unit/credit_card_methods_test.rb

```diff
@@ -128,9 +128,41 @@ def test_should_detect_diners_club_dk
   end
 
   def test_should_detect_jcb_cards
+    # 35280000 - 35899999
+    assert_not_equal 'jcb', CreditCard.brand?('3527999900000000')
     assert_equal 'jcb', CreditCard.brand?('3528000000000000')
-    assert_equal 'jcb', CreditCard.brand?('3580000000000000')
-    assert_equal 'jcb', CreditCard.brand?('3088000000000017')
```

Comment:
NOTE: I removed these two lines as I updated the tests to follow a pattern of:

1. Assert not equal for range minimum - 1
2. Assert equal for range minimum
3. Assert equal for range maximum
4. Assert not equal for range maximum + 1

The suite still passes without these lines removed.